src/pages/project/project_wizard/ProjectDescriptionForm.tsx

```diff
@@ -73,6 +75,11 @@ export const ProjectDescriptionForm = ({ thisIsRekProject }: ProjectDescriptionF
                 rows="4"
                 data-testid={dataTestId.projectWizard.descriptionPanel.scientificSummaryNorwegianField}
                 label={t('project.scientific_summary_norwegian')}
+                slotProps={{
+                  htmlInput: { maxLength: MAX_SUMMARY_LENGTH },
+                  formHelperText: { sx: { textAlign: 'end' } },
+                }}
+                helperText={`${field.value?.length ?? 0}/${MAX_SUMMARY_LENGTH}`}
```

Comment:
Disse kunne vært et objekt siden det er det samme på alle fire felter, men i såfall er det mye flere props i dette tekstfeltet som kunne vært samlet på ett sted, og lurer på om det blir mindre oversiktlig å måtte gå til et annet objekt for å finne basic informasjon om hva som sendes inn? Eventuelt kunne jeg laget en descriptionInput-komponent som har alt som er felles og tar inn det som varierer da?